### PR TITLE
Tweak: Renamed Elementor's responsive SCSS vars (ED-3969)

### PR DESCRIPTION
**Pull request by udidol**
Changelog:
* Tweak: Renamed Elementor's responsive SCSS variables (`$screen-sm/md/lg-X`) to match their intended device names (`$screen-mobile/tablet/desktop-X`).

### CODE REVIEW

File: assets/dev/scss/frontend/breakpoints/breakpoints.scss [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 @import "proxy";
 // End
 
-$screen-lg-max: map_get($breakpoints, xl) - 1;
+$screen-desktop-max: map_get($breakpoints, xl) - 1;
 $screen-xl-min: map_get($breakpoints, xl);
 $screen-xl-max: map_get($breakpoints, xxl) - 1;
-$screen-xxl-min: map_get($breakpoints, xxl);
\ No newline at end of file
+$screen-xxl-min: map_get($breakpoints, xxl);
```
Comment (kobizz) [inline assets/dev/scss/frontend/breakpoints/breakpoints.scss]: Remove this file and its usage

File: assets/dev/scss/frontend/breakpoints/breakpoints.scss [hunk 2]
```diff
@@ -1,11 +1,11 @@
 $screen-xs-max: map_get($breakpoints, sm) - 1;
-$screen-sm-min: map_get($breakpoints, sm);
+$screen-mobile-min: map_get($breakpoints, sm);
```
Comment (kobizz) [inline assets/dev/scss/frontend/breakpoints/breakpoints.scss:2]: Move to `values.scss`

File: assets/dev/scss/frontend/breakpoints/breakpoints.scss [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 
 // Start editable breakpoints area
 @import "proxy";
 // End
 
-$screen-lg-max: map_get($breakpoints, xl) - 1;
+$screen-desktop-max: map_get($breakpoints, xl) - 1;
```
Comment (kobizz) [inline assets/dev/scss/frontend/breakpoints/breakpoints.scss:8]: Move to `values.scss`

File: assets/dev/scss/frontend/breakpoints/breakpoints.scss [hunk 4]
```diff
@@ -1,11 +1,11 @@
 $screen-xs-max: map_get($breakpoints, sm) - 1;
```
Comment (kobizz) [inline assets/dev/scss/frontend/breakpoints/breakpoints.scss:1]: Remove this variable and use a hardcoded value when it's needed

**Review by kobizz — CHANGES_REQUESTED** [review]

File: assets/dev/scss/frontend/breakpoints/breakpoints.scss (hunk 2 above)
Comment (udidol) [reply assets/dev/scss/frontend/breakpoints/breakpoints.scss]: Done.

File: assets/dev/scss/frontend/breakpoints/breakpoints.scss (hunk 1 above)
Comment (udidol) [reply assets/dev/scss/frontend/breakpoints/breakpoints.scss]: Done.

File: assets/dev/scss/direction/frontend-msie.scss [hunk 5]
```diff
@@ -31,10 +31,10 @@ $direction: ltr;
 		}
```
[the remark this answers is not in the document]
Comment (udidol) [reply assets/dev/scss/direction/frontend-msie.scss]: Done.

File: assets/dev/scss/editor/_breakpoints.scss [hunk 6]
```diff
@@ -1,12 +1,20 @@
 $editor-screen-xs-max: map_get($breakpoints, sm) - 1;
```
[the remark this answers is not in the document]
Comment (udidol) [reply assets/dev/scss/editor/_breakpoints.scss]: Done.

File: assets/dev/scss/frontend/breakpoints/breakpoints.scss (hunk 3 above)
Comment (udidol) [reply assets/dev/scss/frontend/breakpoints/breakpoints.scss]: Done.

File: assets/dev/scss/frontend/breakpoints/breakpoints.scss (hunk 4 above)
Comment (udidol) [reply assets/dev/scss/frontend/breakpoints/breakpoints.scss]: Done.

**Pull request merged**